### DISCUSSION TITLE
Make pubsub channel size configurable

### DISCUSSION THREAD

**Discussion by andrii-zhuk** [aembke/fred.rs#125] [Ideas]
As I can see in the code (https://github.com/aembke/fred.rs/blob/main/src/modules/inner.rs#L81) pubsub is a broadcast channel with fixed default capacity of 32. I am hitting channel overflow error quite often when I try to receive messages.

Is it possible to increase this value, or ideally make it configurable?

**Comment by aembke** [comment]
Have you tried 6.1.0? I added a global function to control this (shown in the linked code there), but it needs to be called before allocating the client with `RedisClient::new`.
